ext/rotoscope/rotoscope.c

```diff
@@ -198,7 +196,7 @@ static void log_stack_frame(FILE *stream, rs_stack_t *stack, rs_tracepoint_t tra
   }
   else if (event & EVENT_RETURN)
   {
-    if (trace_pops_stack(trace, stack))
+    if (tracecmp(&trace, &stack_peek(stack)->tp))
```

Comment:
Remember, `stack_peek` _could_ return `NULL` now (theoretically it should never do so), so this could crash. Maybe `stack_peek` should panic (i.e., `exit`)?